src/app/base/components/FilterAccordion/FilterAccordion.tsx

```diff
@@ -23,6 +23,7 @@ type FilterValues = Map<FilterValue, number>;
 type FilterSections = Map<FilterKey, FilterValues>;
 
 export type Props<I, PK extends keyof I> = {
+  ariaLabel?: string;
```

Comment:
While all DOM attributes are camelCased in React, the `aria-` are an exception. II'd suggest to stick to the convention and use a standard aria attribute with a hyphen (`aria-label`). 

Even better if you use the standard aria type from react so that it's always in sync with what React expects even if that changes in future versions. You can do that by extending AriaAttributes from React, e.g.:

```tsx
import type { AriaAttributes } from "react";
export type Props<I, PK extends keyof I> = {
 ...
} & Pick<AriaAttributes, "aria-label">;
```

Reply:
More on this in react accessibility docs: https://reactjs.org/docs/accessibility.html#wai-aria